Makefile

```
@@ -1,19 +1,39 @@
+GOLANG_VERSION := '1.20'
+RUBY_VERSION := '3.3.4'
+GOLANGCI_LINT_VERSION := '1.60.1'
+
 fmt:
-	docker run -v $(shell pwd):/usr/src/plugin -v $(shell pwd)/.build:/go -w /usr/src/plugin golang:1.20 go fmt
+	docker run -v $(shell pwd):/usr/src/plugin -v $(shell pwd)/.build:/go -w /usr/src/plugin golang:${GOLANG_VERSION} go fmt
 
 lint:
-	docker run --rm -v $(shell pwd):/app -w /app golangci/golangci-lint:v1.60.1 /bin/bash -c "go mod vendor && golangci-lint run -v --modules-download-mode vendor --timeout=10m"
+	docker run --rm -v $(shell pwd):/app -w /app golangci/golangci-lint:v${GOLANGCI_LINT_VERSION} /bin/bash -c "git config --global --add safe.directory /app && go mod vendor && golangci-lint run -v --modules-download-mode vendor --timeout=10m"
```

Tested the following locally (on my setup where I managed to repro the behavior from CI) and that use of `GOFLAGS` seems to do the trick:
```suggestion
	docker run --rm -v $(shell pwd):/app -w /app -e GOFLAGS="-buildvcs=false" golangci/golangci-lint:v${GOLANGCI_LINT_VERSION} /bin/bash -c "go mod vendor && golangci-lint run -v --modules-download-mode vendor --timeout=10m"
```

---

Ah, I tried it when I read the [comment above](https://github.com/Automattic/packer-plugin-hostmgr/pull/3#discussion_r1720275371): f4600464bdb5b8e0f000c289fef78cbdd4403af7 and it [worked fine](https://buildkite.com/automattic/hostmgr-packer-plugin/builds/13)! 🥳